src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(element => {
```

Comment:
it's rather an action, not just an element
```suggestion
  actions.forEach(action => {
```